content/recipes/introduction/priorization.md

```diff
@@ -0,0 +1,207 @@
+(fcb-prioritization)=
+# Prioritization of projects for FAIRification
+
++++
+<br/>
+
+````{panels_fairplus}
+:identifier_text: FCB055
+:identifier_link: 'https://w3id.org/faircookbook/FCB055'
+:difficulty_level: 2
+:recipe_type: guidance
+:reading_time_minutes: 10
+:intended_audience:   
+:has_executable_code: nope
+:recipe_name: Prioritization of projects for FAIRification
+```` 
+
+
+## Main Objectives 
+
+The main purpose of this recipe is:
+
+> To provide an overview of elements that could help users in prioritisation and selection of potential projects, processes, or data for FAIRification. For each of these elements, we provide comparative benefits between different stages and justify the order of the stages required for FAIRification. Finally, we also highlight the reasons for making the data FAIR.
+
+---
+## Introduction
+With the increased awareness in the FAIR principles, the drive to implement them can be felt in projects and programs. However, considering the volume and variety of such projects and the finite nature of available resources, it is also necessary to establish a principle approach to prioritizing project or data FAIRification. The present recipe aims to provide insights into how to go about this by outlining essential elements which have been considered and �battle tested�.  
+
+In the recipe, is structured in the following way:
+* We would begin by pointing out the reasons why one needs to make their dataset FAIR. 
+* Following this, a brief overview of the dataset corpus formulation. Once the team has collected all the necessary data, it can now pass through the prioritisation phase. Here a thorough assessment of the dataset based on certain criteria is done. This assessment in-turn leads to development of a scorecard.
```

Comment:
```suggestion
* Following this, a brief overview of the dataset corpus formulation. Once the team has collected all the necessary data, it can now pass through the prioritization phase. Here, a thorough assessment of the dataset based on certain criteria is done. This assessment, in turn, leads to the development of a scorecard.
```